sema/type.go

```diff
@@ -2076,11 +2076,15 @@ type ArrayType interface {
 	isArrayType()
 }
 
+const ArrayTypeFirstIndexFunctionName = "firstIndex"
```

Comment:
👌 

Thank you for addressing this tech-debt! In a follow up PR we can maybe use these in the interpreter

Reply:
I think it's already used in the interpreter as well.